lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -668,23 +678,29 @@ export class UserAgentApplication {
                 request.correlationId,
             );
 
+            this.logger.verbose(`ServerAuthenticationRequest: ${JSON.stringify(serverAuthenticationRequest)}`);
```

Comment:
I think providing succinct, specific messages will be more helpful than stringify entire objects. What sort of specific pieces of information might helpful to know here?

Reply:
Agreed. Changed logger to no longer print out object, but simply state that a server authentication request was built.